model/src/main/proto/interaction_object.proto

```diff
@@ -0,0 +1,59 @@
+syntax = "proto3";
+
+package model;
+
+option java_package = "org.oppia.app.model";
+option java_multiple_files = true;
+
+// Object type of the interaction object
+enum ObjectType {
+  NORMALIZED_STRING = 0;
+  SIGNED_INT = 1;
+  NON_NEGATIVE_INT = 2;
+  REAL = 3;
+  BOOLEAN = 4;
+  NUMBER_WITH_UNITS = 5;
+  SET_OF_HTML_STRING = 6;
+}
+
+// Structure for any interaction object
+message InteractionObject {
+  ObjectType object_type = 1;
+  oneof value {
```

Comment:
Actually, I was wondering if we could call the oneof object_type. Do we need a separate enum? I thought oneof generated an enum for us to switch on & retrieve the value. If that's right, we don't need the extra enum and it's a bit redundant.

Reply:
Ah oops I misunderstood. Changed it